src/dotnet/FileSystemHost.cs

```diff
@@ -194,6 +194,11 @@ public Boolean PassQueryDirectoryFileName
             get { return 0 != (_VolumeParams.Flags & VolumeParams.PassQueryDirectoryFileName); }
             set { _VolumeParams.Flags |= (value ? VolumeParams.PassQueryDirectoryFileName : 0); }
         }
+        public Boolean FlushAndPurgeOnCleanup
+        {
+            get { return 0 != (_VolumeParams.Flags & VolumeParams.FlushAndPurgeOnCleanup); }
+            set { _VolumeParams.Flags |= (value ? VolumeParams.FlushAndPurgeOnCleanup : 0); }
+        }
```

Comment:
Looks good!

Reply:
I agreed the Contributor Agreement and I was under the impression that the updated version (with my name in it) was waiting for your approval. Am I mission something there ?

Reply:
Thank you for your changes. Everything looks in order.

Regarding the Contributor Agreement, I have not received any document on this or any other Pull Request.

The preferred way to do this would be to add your name to the Contributor Agreement in your fork and add it as a commit that will then be added to this Pull Request. When I merge this Pull Request into WinFsp your name will also be included in the Contributor Agreement document.